Sofa/Component/SolidMechanics/Spring/src/sofa/component/solidmechanics/spring/RestShapeSpringsForceField.inl

```diff
@@ -183,14 +201,14 @@ void RestShapeSpringsForceField<DataTypes>::recomputeIndices()
     m_indices.clear();
     m_ext_indices.clear();
 
-    for (sofa::Index i = 0; i < d_points.getValue().size(); i++)
+    for (const sofa::Index i : d_points.getValue())
     {
-        m_indices.push_back(d_points.getValue()[i]);
+        m_indices.push_back(i);
     }
 
-    for (sofa::Index i = 0; i < d_external_points.getValue().size(); i++)
+    for (const sofa::Index i : d_external_points.getValue())
     {
-        m_ext_indices.push_back(d_external_points.getValue()[i]);
+        m_ext_indices.push_back(i);
```

Comment:
Not technically required for the PR, but I really wanted to clean it up a little. :D